pkgs/tools/security/doppler/default.nix

```
@@ -0,0 +1,31 @@
+{ buildGoModule, fetchFromGitHub, stdenv }:
```

preference to avoid stdenv
```suggestion
{ buildGoModule, fetchFromGitHub, lib }:
```